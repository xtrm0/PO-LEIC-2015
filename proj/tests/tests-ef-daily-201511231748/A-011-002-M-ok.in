2
doc2
8
4
1
2
0
4
2
2
0
0
0
