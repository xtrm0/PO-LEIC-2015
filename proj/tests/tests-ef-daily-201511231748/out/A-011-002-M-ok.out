Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Ficheiro a abrir: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Identificador da secção: Nova secção activa: '1'.
Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: [] {Sub 1.1}
[] {Sub 1.2}
Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Identificador da secção: Nova secção activa: '2'.
Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: [] {Sub 2.1}
[] {Sub 2.2}
Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Menu de Edição
1 - Mudar título
2 - Listar secções
3 - Mostrar conteúdo
4 - Seleccionar secção
5 - Inserir secção
6 - Nomear secção
7 - Remover secção
8 - Inserir parágrafo
9 - Nomear parágrafo
10 - Alterar parágrafo
11 - Remover parágrafo
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: 